combat-trainer.lic

```diff
@@ -1998,6 +2013,177 @@ class TrainerProcess
     DRRoom.dead_npcs.size > dead_count
   end
 
+  def check_heavens(game_state)
+    @equipment_manager.stow_weapon(game_state.weapon_name)
+    echo "Getting telescope"
+    case DRC.bput("get my telescope","You get a", "You need a free hand to pick that up.", "You are already holding that.")
+    when "You get a"
+      determine_time
+    when "You need a free hand to pick that up."
+      quick_retry
```

Comment:
❓ Where is `quick_retry` defined?

Reply:
quick_retry should not have been included in the combat-trainer version. That's left over from walkingastro.lic. Will fix.